features/step_definitions/batch_action_steps.rb

```diff
@@ -20,8 +20,7 @@
 end
 
 Then /^I (should|should not) see the batch action (button|selector)$/ do |maybe, type|
-  selector = "div.table_tools .batch_actions_selector"
-  selector << " .dropdown_menu_button" if maybe == "should" && type == "button"
+  selector = maybe == "should" && type == "button" ? "div.table_tools .batch_actions_selector .dropdown_menu_button" : "div.table_tools .batch_actions_selector"
```

Comment:
I think just changing `<<` in both places in this file is easier to read requires no duplication of class names.

Reply:
Unfortunately, can't use `<<` because it mutates the string but can use `+=` which will read similarly.

Reply:
Sure, yeah, I meant changing `<<` to `+=` indeed!